charts/consul/templates/server-config-configmap.yaml

```diff
@@ -187,4 +187,29 @@ data:
       }
     }
   {{- end }}
+  {{- if .Values.server.auditLogs.enabled }}
+  audit-logging.json: |-
+    {
+      "audit": {
+        "enabled": "true",
+        "sink": {
```

Comment:
sink seems to be a map of `AuditSinks` according to https://github.com/hashicorp/consul/blob/88951bfafaa969f52235dc5b443cb64ba329660c/agent/config/config.go#L811 and so we need to iterate over the key value pair list

Reply:
we can remove `countOuter` and `countInner` and make it more pragmatic by this 

```
 {
      "audit": {
        "enabled": "true",
        "sink": {
          {{- range $index, $element := .Values.server.auditLogs.sinks }}
          {{- if ne $index 0 }},{{end}}
          "{{ $element.name }}": {
            {{- $firstKeyValuePair := true }}
            {{- range $k, $v := . }}
              {{- if ne $k "name" }}
              {{- if ne $firstKeyValuePair true }},{{end}}
              {{- $firstKeyValuePair = false}}
              "{{ $k }}": "{{ $v }}"
              {{- end }}
            {{- end }}
          }
          {{- end }}
        }
      }
    }
```